vspec/vspec2x.py

```diff
@@ -38,6 +40,7 @@ def main(self, arguments):
         initLogging()
         parser = argparse.ArgumentParser(description="Convert vspec to other formats.")
 
+        parser.add_argument('-v', '--version', action='version', version=VERSION)
```

Comment:
-v is often used for --verbose

Consider using -V for --version or not defining a short-form option?